projects/packages/publicize/src/class-publicize.php

```diff
@@ -53,9 +53,7 @@ public function __construct() {
 		add_filter( 'jetpack_sharing_twitter_via', array( $this, 'get_publicized_twitter_account' ), 10, 2 );
 
 		add_action( 'updating_jetpack_version', array( $this, 'init_refresh_transient' ) );
-
-		include_once __DIR__ . '/enhanced-open-graph.php';
-
+		include_once __DIR__ . '/../../../../plugins/jetpack/modules/publicize/enhanced-open-graph.php';
```

Comment:
We should probably take the `enhanced-open-graph.php` out of the package altogether: Related: p1649163773697909/1649159331.678819-slack-C02JJ910CNL